src/criteria_assessment/criteria.jl

```diff
@@ -211,6 +225,9 @@ function setup_region_routes(config, auth)
             )
         )
 
+        # Specifically clear from memory to invoke garbage collector
+        assessed = nothing
```

Comment:
shouldn't be necessary, `assessed` is local to this function. Did you actually see this make a difference?

Reply:
Yes, yes it does make a difference.

Triggers the GC to run a bit earlier, making space for the next big dataset. Otherwise you're more likely to hit an out-of-memory error.